docs/build/setting-up/set-up-development-env.md

```diff
@@ -74,4 +75,4 @@ For advanced users: you can override marine and mrepl version in your project us
 
 ### Docker
 
-You will need that if you want to work with a local Fluence Network. We recommend to install Docker Desktop for yor OS using [official Docker documentation](https://docs.docker.com/desktop/). Don't forget to run docker after installation, before you attempt to set up local Fluence Network using `fluence local up`
+You need Docker if you want to work with a local Fluence Network, which comes in rather handy when testing your code before deploying it to the remote dar or kras environments. We recommend to install Docker Desktop for your OS using [official Docker documentation](https://docs.docker.com/desktop/). Don't forget to run docker after installation, before you attempt to set up local Fluence Network using `fluence local up`
```

Comment:
```suggestion
If you want to run a local Fluence Network to test your code before deploying it to the remote dar or kras, you need to install Docker Desktop for your OS using [official Docker documentation](https://docs.docker.com/desktop/). Don't forget to run the docker daemon after installation and  before you set up a local Fluence Network using `fluence local up`.
```

Reply:
you do not need Docker Desktop per se, you just need docker. That's why I mentioned that we recommed installing Docker Desktop as the simplest way to interact with docker